packages/swap/tests/okx.test.ts

```
@@ -0,0 +1,554 @@
+} from "../src/types";
+import BN from "bn.js";
+import { isValidSolanaAddressAsync } from "../src/utils/solana";
+import {
+  fromToken,
+  amount,
+  fromAddress,
+  toAddress,
+  toToken,
+  nodeURL,
+} from "./fixtures/solana/configs";
+import { isValidSolanaAddressAsync } from "../src/utils/solana";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'isValidSolanaAddressAsync'. Consider to delete it or rename it.

'isValidSolanaAddressAsync' is defined here:


(lint/suspicious/noRedeclare)